samples/CrawlFeaturizer/Util/CognitiveTextAnalyzer.cs

```diff
@@ -1,62 +1,43 @@
-﻿using Microsoft.Azure.CognitiveServices.Language.TextAnalytics;
-using Microsoft.Azure.CognitiveServices.Language.TextAnalytics.Models;
+﻿using Azure;
+using Azure.AI.TextAnalytics;
 using System.Collections.Generic;
-using System.Linq;
 using System.Threading.Tasks;
 
 namespace CrawlFeaturizer.Util
 {
     public class CognitiveTextAnalyzer
```

Comment:
Now that there is no `ITextAnalyticsClient` this class shouldn't be needed. The code in `Program.cs` can directly call the functions in the `TextAnalyticsClient`